sdk/nodejs/cmd/pulumi-language-nodejs/main.go

```diff
@@ -78,6 +79,9 @@ const (
 	// PULUMI_LANGUAGE_NODEJS_RUN_PATH, which we do in some testing cases.
 	defaultRunPath = "@pulumi/pulumi/cmd/run"
 
+	// The path to run a pluginn.
```

Comment:
```suggestion
	// The path to run a plugin.
```
What does this mean though? This looks like a module import whereas the comment sorta feels like we are talking about a CWD-ish thing -- what will it be used for?

Reply:
It's the path to the thing that effectively launches the plugin.  I'll rewrite the comment.